src/test/java/CommandReminderTest.java

```diff
@@ -0,0 +1,28 @@
+import duke.command.CommandReminder;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.TaskType;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class CommandReminderTest {
+
+    @Test
+    void execute() {
+        TaskList taskList = new TaskList();
+        Task testTask = TaskList.createTask(TaskType.EVENT,"something/by somewhen");
+        taskList.addTask(testTask);
+        CommandReminder testCommand = new CommandReminder();
+        testCommand.execute(taskList);
+
+        Task mainTask = taskList.getList().get(0);
+        assertEquals(true, mainTask.isQueuedTasks());
+//        Task queuedTask = mainTask.getQueuedTasks().getList().get(0);
+//        assertEquals(TaskType.EVENT, queuedTask.getTaskType());
+//        assertEquals("Ice Cream Party", queuedTask.getTaskName());
+//        assertEquals("", queuedTask.getDetailDesc());
+//        assertEquals("Tomorrow", queuedTask.getTaskDetails());
```

Comment:
The rest is just me testing whether a certain condition is true or not. _assertEquals_ is a JUnit function that is essentially '==' but in a method call format. 

So basically what I'm doing here is; 
if (TaskType.Event == queuedTask,getTaskType) {
   return;
} else {
   throw Assert Error;
}

Reply:
The first argument is the thing you expect it to be, the second is the thing you're testing against.